docs/samples/Microsoft.ML.Samples/Dynamic/LightGBM/LightGBMMulticlassClassification.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML.Data;
+using Microsoft.ML.SamplesUtils;
+
+namespace Microsoft.ML.Samples.Dynamic.LightGBM
+{
+    class LightGbmMulticlassClassification
+    {
+        public static void LightGbmMulticlassClassificationExample()
+        {
+            // Create a general context for ML.NET operations. It can be used for exception tracking and logging,
+            // as a catalog of available operations and as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Create in-memory examples as C# native class.
+            var examples = DatasetUtils.GenerateRandomMulticlassClassificationExamples(1000);
+
+            // Convert native C# class to IDataView, a consumble format to ML.NET functions.
+            var dataView = mlContext.Data.ReadFromEnumerable(examples);
+
+            // Create a pipeline. 
+            //  - Convert the string labels into key types.
+            //  - Apply LightGbm multiclass trainer
+            var pipeline = mlContext.Transforms.Conversion.MapValueToKey("LabelIndex", "Label")
+                        .Append(mlContext.MulticlassClassification.Trainers.LightGbm(labelColumn: "LabelIndex"))
+                        .Append(mlContext.Transforms.Conversion.MapValueToKey("PredictedLabelIndex", "PredictedLabel"))
+                        .Append(mlContext.Transforms.CopyColumns("Scores", "Score"));
+
+            // Split the static-typed data into training and test sets. Only training set is used in fitting
+            // the created pipeline. Metrics are computed on the test.
+            var (trainingData, testingData) = mlContext.MulticlassClassification.TrainTestSplit(dataView, testFraction: 0.5);
+
+            // Train the model.
+            var model = pipeline.Fit(trainingData);
+
+            // Do prediction on the test set.
+            var dataWithPredictions = model.Transform(testingData);
+
+            // Evaluate the trained model is the test set.
+            var metrics = mlContext.MulticlassClassification.Evaluate(dataWithPredictions, label: "LabelIndex");
+
+            // Check if metrics are resonable.
+            Console.WriteLine("Macro accuracy: {0}, Micro accuracy: {1}.", 0.863482146891263, 0.86309523809523814);
+
+            // Convert prediction in ML.NET format to native C# class.
```

Comment:
>prediction in ML.NET format to native C# class. [](start = 23, length = 47)

IDataView with predictions, to an IEnumerable<DatasetUtils.MulticlassClassificationExample>